### ISSUE TITLE
Go through 508 Checklist

### ISSUE THREAD

**Issue by noahmanger** [18F/openFEC#659]

**Comment by msecret** [comment]
From Nick Bristow:

```
http://fec-dev-web.cf.18f.us/
Skip-nav link is needed --global

Placeholder text in search boxes needs to be darker --global

use aria-describedby for the glossary search. The id should be the results--global

http://fec-dev-web.cf.18f.us/candidates

Filters on left, the inputs need ID’s that match their labels for attributes

Labels on the filter form have insufficient contrast

The blue name text on the dark rows has insufficient contrast
```
the ? help links doesn't auto populate if you select with keyboard. Good job with the accessibility stuff, i'm really impressed :smile:

**Comment by msecret** [comment]
Everything but contrast problems were fixed with https://github.com/18F/openFEC-web-app/pull/133 and https://github.com/18F/openFEC-web-app/pull/132
